.github/CONTRIBUTING.md

```diff
@@ -180,12 +182,14 @@ In case you've got a small change in most of the cases, your pull request would
 -   Write tests
 -   Follow the existing coding style
 -   Write a [good commit message](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html)
+-   Make sure your PR's description contains GitHub's special keyword references that automatically close the related issue when the PR is merged. ([More info](https://github.com/blog/1506-closing-issues-via-pull-requests))
```

Comment:
I would add a new point: "make sure your PR as an issue, if it doesn't, please create one. This would help discussion with the community, and polishing ideas in case of a new feature."